Applications/ReinforcementLearning/DeepQ/jni/meson.build

```diff
@@ -32,4 +32,4 @@ e = executable('nntrainer_deepq',
   install_dir: application_install_dir
 )
 
-test('app_DeepQ', e, args: [res_path / 'DeepQ.ini'], timeout: 60)
+# test('app_DeepQ', e, args: [res_path / 'DeepQ.ini'], timeout: 60)
```

Comment:
Any reason of this?

Reply:
DeepQ started to fail like 100 commits ago, it needs fix but will take time. So commented for now.

Reply:
> DeepQ started to fail like 100 commits ago, it needs fix but will take time. So commented for now.

Please make issue for this.